sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/bigquery/BeamBigQuerySqlDialect.java

```diff
@@ -156,7 +156,14 @@ public void unparseCall(
         break;
       case OTHER_FUNCTION:
         String funName = call.getOperator().getName();
-        if (FUNCTIONS_USING_INTERVAL.contains(funName)) {
+        if (funName.equals("numeric_literal")) {
+          // self-designed function
```

Comment:
Also, could you move this logic to a function `unparseNumericLiteralWrapperFunction()` like we do in other branches?

Reply:
Done. Thanks for your opinion.